composer.json

```diff
@@ -21,18 +21,15 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.3.0 | ^8.0",
-        "ramsey/uuid" : "^3.6.0 || ^4.0",
+        "php": "^7.4 || ^8.0",
+        "ramsey/uuid": "4.2.1 || ^4.3",
```

Comment:
4.2.1 is used for php74+lowest. setting it to ~4.2.1 would also use a incompatible version in php81+lowest